frontend/src/components/user-management/student/AddClassroomModal.tsx

```diff
@@ -102,7 +123,27 @@ const AddClassroomModal = (): React.ReactElement => {
       setRequestErrorMessage(
         "Please ensure all required components are filled out before submitting your application.",
       );
+      return;
     }
+    const classObj: ClassRequest = {
```

Comment:
can we use `handleSubmit` here? then we can use the `data` object instead of having all the useState fields (example code [here](https://github.com/uwblueprint/jump-math/blob/ca65dc5688234aa8e037f89b29f149db32c834f2/frontend/src/components/pages/admin/CreateAssessmentPage.tsx#L52))